source/_static/css/frc-rtd.css

```diff
@@ -123,6 +123,20 @@
 	}
 }
 
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	/*            Apple           Chrome macOS      Win >= Vista   Android + ChromeOS,  KDE          Guess   Win <= XP    old android   backup     default      apple emojis         win emojis        wingdings?*/
+}
+
+.rst-content .toctree-wrapper>p.caption, h1, h2, h3, h4, h5, h6, legend {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
```

Comment:
If we keep "Segoe UI". We should increase weight of the font when inside headers and the sidebar to something thicker, perhaps `600`

![image](https://user-images.githubusercontent.com/10674555/134013969-6e978328-7d25-489a-a86e-813f5553a90e.png)
vs
![image](https://user-images.githubusercontent.com/10674555/134014010-eaaba044-123f-4515-8061-157dde312172.png)